src/views/CountView.vue

```diff
@@ -0,0 +1,23 @@
+<template>
```

Comment:
Would it not make more sense to have a ParticipantCount component ( not View ) and wire it to the store from the sidebar View ?

Reply:
That's how I started out but I found it was quite a bit of wiring and a teenytiny component.

Reply:
Just wire from the side bar , it makes no difference

Reply:
I think it does make a difference, it makes the sidebar bigger and harder to test.
I really liked the changes I got when introducing the view.
Would you like me to split off a Count component from the view?